parser/spdx/spdx_test.go

```diff
@@ -0,0 +1,33 @@
+package spdx
+
+import "testing"
+
+func TestSPDXParser(t *testing.T) {
+	p := `SPDXVersion: SPDX-2.2
+DataLicense: conftest-demo
+SPDXID: SPDXRef-DOCUMENT
+DocumentName: hello
+DocumentNamespace: https://swinslow.net/spdx-examples/example1/hello-v3
+Creator: Person: Steve Winslow (steve@swinslow.net)
+Creator: Tool: github.com/spdx/tools-golang/builder
+Creator: Tool: github.com/spdx/tools-golang/idsearcher
+Created: 2021-08-26T01:46:00Z
+`
+
+	parser := &Parser{}
+
+	var input interface{}
+	if err := parser.Unmarshal([]byte(p), &input); err != nil {
+		t.Fatalf("parser should not have thrown an error: %v", err)
+	}
+
+	if input == nil {
+		t.Error("There should be information parsed but its nil")
+	}
+
+	inputMap := input.(map[string]interface{})
+	creationInfo := inputMap["CreationInfo"].(map[string]interface{})
+	if creationInfo["DataLicense"] != "conftest-demo" {
+		t.Error("DataLicense of parsed SPDX file's should be a conftest-demo, but was not")
```

Comment:
nit: Unit test errors should include a `have: X, want: Y` message to reduce the amount of time troubleshooting failed tests.